### DISCUSSION TITLE
Mode 9 does not unload

### DISCUSSION THREAD

**Discussion by nitroSkirata** [Courseplay/courseplay#7408] [Q&A] [answered]
Hey,

I try to use mode 9 to empty a silo into the BGP Bunker.

I am pretty sure, that i followed the instructions for mode 9 correctly, but the Loader does not stop and/or dump at the waitpoint, it simply goes into reverse with a full shovel.

Could someone please tell me, what i am doing wrong?

here is a vid of the incident: https://www.youtube.com/watch?v=1z38AeCU--I

Kind regards,
JDTiger

**Comment by Tensuko** [comment]
What Version do you use ?
I would like to see where the Trigger of your BGA Bunker is (F5) and the debug Informations (Channel 9 and 10).
So I can see the Check of the Trigger.

The Direction of your Waitpoint looked correct.

**Reply by nitroSkirata** [reply]
rgr, here a vid with channels 9&10 as well as the F5 view:
https://youtu.be/2A6t3ikHHOk

Version used is the current dev version 6.4.0.5

**Comment by Tensuko** [comment, marked as answer]
Hm,
That is strange...
Use the Editor and move the waitpoint arround, you will notice that the Arrow of the Waypoint will look at the next waypoint, you can have a normal waypoint behind the waitpoint, just make sure it does look at the Trigger.
Normaly it should happen automatically, but somehow I get the feeling it could not find your BGA Trigger, else it is to close, or it does look in the wrong direction.

Look at what I did there:
https://youtu.be/qCBUuo6Z2e8?t=120

**Reply by nitroSkirata** [reply]
Yep, pulling the waitpoint back a fair bit, was what made it work, thanks for the help...

I was mainly wondering, why it didn't stop, like it did in your vid, when it couldn't find the trigger
